set.mm

```diff
@@ -481233,6 +481329,394 @@ such that every prime ideal contains a prime element (this is a
+        DXKXLWPWSWNEAUNUOXDXMXJWTRZXKXDXJAOZWSAOXMXNTXOXDWNEAUPVAWSAUQXJWSAURVB
+        XJWTVCVTVDVEVFDWLXJWTGLWNEAABVGKVHZVIVJVKVLXHXDXHSXGXHXDWRXBWSWPXBMZDWL
+        VMWRXBOXQDWLXQWPAOWOEAUPWPAXPVNVOVPDWLWPXBWQWQQZVQVRWAVSXHXFXDWSWBMXHXF
+        THWCDWLWPWSWQWBXRWDVRWEWFXAHXBWGWHWIWJ $.
+
+      $d P x y $.  $d P z $.  $d R i s x y $.  $d R i z $.  $d R j $.
+      $d V i s x y $.  $d V i z $.
+      $( The Zariski topology is a topology, and its closed sets are images by
+         ` V ` of the ideals of ` R ` .  (Contributed by Thierry Arnoux,
+         16-Jun-2024.) $)
+      zartopn $p |- ( R e. CRing -> ( J e. ( TopOn ` P ) /\ ran V = ( Clsd ` J
+        ) ) ) $=
+        ( vs wcel cfv ccld wceq wa cv crab eqid vx ccrg ctopon crn cdif cpw wss
+        vy vz clidl wral ssrab2 cprmidl fvexi elpw2 mpbir rgenw rnmptss a1i crg
+        ax-mp crngring c0g csn cmpt rabeqi mpteq2i eqtri zarcls0 wfun cdm lidl0
+        funmpt2 rabex dmmpti eleqtrrdi fvelrn sylancr eqeltrrd zarclsint ismred
+        syl cbs c0 lidl1 eleqtrdi zarcls1 mpbiri mpdan syl2anc zarclsun mretopd
+        zarcls eleq1d fveq2d eqeq2d anbi12d mpbird ) BUBMZFAUCNZMZGUDZFONZPZQAL
+        RUEXBMLAUFZSZWTMZXBXFONZPZQWSUAUHLAXFXBWSXBAUIXBXEUGZWSDRERUGZEASZXEMZD
+        BUJNZUKXJXMDXNXMXLAUGXKEAULXLAABUMJUNZUOUPUQDXNXLXEGKURVAUSWSBUTMZAXBMB
+        VBZXPBVCNZVDZGNZAXBABDEGXRGDXNXLVEDXNXKEBUMNZSZVEKDXNXLYBXKEAYAJVFVGVHZ
+        JXRTZVIXPGVJZXSGVKZMXTXBMDXNXLGKVMZXPXSXNYFBXNXRXNTYDVLDXNXLGXKEAXOVNKV
+        OZVPXSGVQVRVSWBBUIRDEGYCVTWAWSBWCNZGNZWDXBWSYIXNMZYJWDPZWSYIYFXNWSXPYIY
+        FMZXQYIBYFYHYITZWEWBZYHWFWSYKQYLYIYIPYNYIBDEYIGYCYNWGWHWIWSYEYMYJXBMYEW
+        SYGUSYOYIGVQWJVSBDEGUARUHRYCWKXFTWLWSXAXGXDXIWSFXFWTWSXPFXFPXQABCDEFGLH
+        IJKWMWBZWNWSXCXHXBWSFXFOYPWOWPWQWR $.
+    $}
+
+    $d R i j k $.
+    $( The Zariski topology is a topology.  (Contributed by Thierry Arnoux,
+       16-Jun-2024.) $)
+    zartop $p |- ( R e. CRing -> J e. Top ) $=
+      ( vi vj vk ccrg wcel cprmidl cfv ctopon ctop clidl cv wss crab cmpt wceq
+      crn ccld eqid sseq1 rabbidv cbvmptv zartopn simpld topontop syl ) AIJZCAK
+      LZMLJZCNJUKUMFAOLZFPZGPZQZGULRZSZUACUBLTULABHGCUSDEULUCFHUNURHPZUPQZGULRU
+      OUTTUQVAGULUOUTUPUDUEUFUGUHULCUIUJ $.
+
+    ${
+      $d P i j k $.
+      zartop.3 $e |- P = ( PrmIdeal ` R ) $.
+      $( The base set of the Zariski topology.  (Contributed by Thierry Arnoux,
+         16-Jun-2024.) $)
+      zartopon $p |- ( R e. CRing -> J e. ( TopOn ` P ) ) $=
+        ( vi vj vk ccrg wcel ctopon cfv clidl cv wss crab cmpt wceq crn rabbidv
+        ccld sseq1 cbvmptv zartopn simpld ) BKLDAMNLHBONZHPZIPZQZIARZSZUADUCNTA
+        BCJIDUMEFGHJUHULJPZUJQZIARUIUNTUKUOIAUIUNUJUDUBUEUFUG $.
+    $}
+
+    $d J d x y $.  $d R d j k $.  $d R i j x y $.
+    $( The Zariski topology is T_0 .  (Contributed by Thierry Arnoux,
+       16-Jun-2024.) $)
+    zart0 $p |- ( R e. CRing -> J e. Kol2 ) $=
```

Comment:
"T_0" = "Kol2"? "T_0" = "Kolmogorov spaces" (see df-t0). Maybe this shoulkd be added here.

Reply:
Yes, $T_0$ spaces are also called [Kolmogorov spaces](https://en.wikipedia.org/wiki/Separation_axiom), and this is formalized as `Kol2` in set.mm.

The right place to explain this is indeed in [df-t0](https://us.metamath.org/mpeuni/df-t0.html). I did not dig about why the "2" in `Kol2`.